pybuildkite/jobs.py

```diff
@@ -16,16 +16,14 @@ def __init__(self, client, base_url):
         self.client = client
         self.path = base_url + "/organizations/{}/pipelines/{}/builds/{}/jobs/{}/"
 
-    def get_job_log(self, organization, pipeline, build, job, get_text=False, get_html=False):
+    def get_job_log(self, organization, pipeline, build, job):
```

Comment:
So instead of removing these we could add the ability to actual specify and use them https://buildkite.com/docs/apis/rest-api/jobs#get-a-jobs-log-output

Reply:
Oops, that's a far better idea. I'll add them in rather than removing them.

Reply:
So I went for a different style to what's listed here, I requested the user provide the Mime they want, this is to avoid the situation where they add a new format in and suddenly we need 3 parameters.

I also kept it optional to ensure that the affordance looks like the API.